.github/workflows/maven-build.yml

```diff
@@ -61,15 +61,16 @@ jobs:
           librdkafka-dev unixodbc-dev libevent-dev
       
       - name: Setup Test Containers Properties
-        run: echo 'testcontainers.reuse.enable=true' >> $HOME/.testcontainers.properties
+        run: |
+          sudo make install_antlr_cli
+          echo 'testcontainers.reuse.enable=true' >> $HOME/.testcontainers.properties
 
       - name: Build PR with Maven
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         if: ${{ github.event_name == 'pull_request_target' }}
```

Comment:
this event only runs if it's pull request target, it will fail as antlr command is under the same condition and if the event is different, we will get antlr4 not found